packages/deploy-script-support/src/externalTypes.js

```diff
@@ -16,22 +16,21 @@ export {};
  */
 
 /**
- * @typedef BundleHandle
- * @property {string} [bundleName]
+ * @typedef {{ bundleName: string } | { bundleID: string} } ManifestBundleRef
```

Comment:
a thought on product interface boundaries...

One consumer (metamask) pointed out that even though the types have no runtime impact, they can represent breaking changes in the sense that ci jobs fail and effort is needed to address them. This type is exported, so renaming it is a breaking change, from that angle. (I suppose changing its definition could be too)

I don't think that should hold up this PR, but I hope we decide soon on a pretty clear set of product interfaces.

Reply:
Good to raise. I queued it for our discussion about breaking changes.